package.json

```diff
@@ -104,6 +104,8 @@
   },
   "dependencies": {
     "@types/react-jsonschema-form": "^1.0.11",
+    "@types/chart.js": "^2.7.40",
```

Comment:
May not have any real consequences, but it's customary to put `@types/*` in `devDependencies` since they're only used at compile-time.

Reply:
Good catch, thanks!